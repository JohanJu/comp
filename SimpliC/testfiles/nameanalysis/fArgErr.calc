int f(int a){
	return a;
}
int main(){
	int b;
	b = f(1,2);
}