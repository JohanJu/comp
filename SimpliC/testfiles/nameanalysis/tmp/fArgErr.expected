Error at line 6: wrong nbr args: f
